api/project/controllers/project.js

```diff
@@ -48,4 +49,73 @@ module.exports = {
     }));
     return sanitizeEntity(entity, {model: strapi.models.project});
   },
+
+  async giveGoogleResources(ctx) {
+    try {
+      const { slug } = ctx.params;
```

Comment:
What's slug?

Reply:
I think it's the projects identifier. It's what mohammod uses to get the projects

Reply:
I see.